build/ProjectReunion-release.yml

```diff
@@ -72,9 +72,9 @@ jobs:
     doCodeSign: 'true'
 
 - job: SignBinariesAndPublishSymbols
-  dependsOn: Build
+  dependsOn: PublishMRT
```

Comment:
>PublishMRT [](start = 13, length = 10)

should also depend on Build

Reply:
I believe Build just builds the Project Reunion sample code project that we aren't using anymore.  We aren't signing or publishing any of the output of Build in signconfig.xml I think, so we should be good.

Reply:
yes, it's ok for now, but it won't be always empty. :)

Reply:
Okay, that makes sense.  I'll have it depend on both.